docs/README.md

```diff
@@ -4,7 +4,7 @@ CLDR main page: [https://unicode.org/cldr](unicode.org/cldr)
 
 ## TR35
 
-- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](ldml/tr35.html)
+- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](https://unicode-org.github.io/cldr/ldml/tr35.html)
```

Comment:
```suggestion
- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](ldml/tr35.md)
```

I don't think we should make this change. This should be a relative link. It's a html link as a compromise until #1572 lands.